tests/ui/parser/labeled-no-colon-expr.stderr

```diff
@@ -77,6 +77,10 @@ LL |     m!({});
    |     ------ in this macro invocation
    |
    = note: this error originates in the macro `m` (in Nightly builds, run with -Z macro-backtrace for more info)
+help: wrap this in another block
+   |
+LL |             'l5 { $b };
```

Comment:
this suggestion is incomplete, but would be applied with the suggestion above ("help: add `:` after the label").